pkg/agent/openflow/client.go

```diff
@@ -374,6 +375,28 @@ type Client interface {
 	// or ip, port, protocol and direction. It is used to bypass NetworkPolicy enforcement on a VM for the particular
 	// traffic.
 	InstallPolicyBypassFlows(protocol binding.Protocol, ipNet *net.IPNet, port uint16, isIngress bool) error
+
+	// GetMeterStats sends a multipart request to get all the meter statistics and
+	// sets values for antrea_agent_ovs_meter_packet_dropped_count.
+	GetMeterStats()
+}
+
+func (c *client) GetMeterStats() {
+	handleMeterStatsReply := func(packetCounts map[int]int64) {
+		for k, v := range packetCounts {
+			switch k {
+			case PacketInMeterIDNP:
+				metrics.OVSMeterPacketDroppedCount.WithLabelValues("PacketInMeterNetworkPolicy").Set(float64(v))
+			case PacketInMeterIDTF:
+				metrics.OVSMeterPacketDroppedCount.WithLabelValues("PacketInMeterTraceflow").Set(float64(v))
+			default:
+				klog.InfoS("Received unexpected meterId", "meterId", k)
```

Comment:
Perhaps make it not logged by default, in case it floods log file once users create their own meters.

Reply:
done